leakcanary-android/build.gradle

```diff
@@ -2,6 +2,7 @@ apply plugin: 'com.android.library'
 
 dependencies {
   api project(':leakcanary-analyzer')
+  implementation 'com.android.support:appcompat-v7:27.1.0'
```

Comment:
You only need `support-compat` for this. AppCompat brings in way too much stuff.

Reply:
The docs say ["belongs to Maven artifact com.android.support:support-compat:28.0.0-alpha1,"](https://developer.android.com/reference/android/support/v4/content/FileProvider.html) but the class is in `support-core-utils`. Is the class moving packages or are the docs wrong?

Reply:
Er, yeah that's what I meant. I used to be in compat before core-utils and core-ui split out of it.

Reply:
Ah, I moved my comment out because it collapsed here. :(

Reply:
I think the docs still need to be updated.

Reply:
It'll be fixed by virtue of [redacted] which will be public soon.